modules/oe_translation_remote/oe_translation_remote.module

```diff
@@ -0,0 +1,125 @@
+<?php
+
+/**
+ * @file
+ * OE Translation remote module file.
+ */
+
+use Drupal\Core\Access\AccessResult;
+use Drupal\Core\Breadcrumb\Breadcrumb;
+use Drupal\Core\Entity\ContentEntityInterface;
+use Drupal\Core\Entity\EntityInterface;
+use Drupal\Core\Link;
+use Drupal\Core\Routing\RouteMatchInterface;
+use Drupal\Core\Session\AccountInterface;
+use Drupal\Core\Url;
+use Drupal\field\Entity\FieldStorageConfig;
+use Drupal\oe_translation_remote\Form\RemoteTranslationReviewForm;
+use Drupal\oe_translation_remote\TranslationRequestRemoteInterface;
+
+/**
+ * Implements hook_entity_type_alter().
+ */
+function oe_translation_remote_entity_type_alter(array &$entity_types) {
+  foreach ($entity_types as $entity_type_id => $entity_type) {
+    if ($entity_type_id === 'oe_translation_request') {
+      /** @var \Drupal\Core\Entity\EntityTypeInterface $remote_translation */
+      $remote_translation = $entity_types[$entity_type_id];
+      $remote_translation->setFormClass('remote_translation_review', RemoteTranslationReviewForm::class);
+      $remote_translation->setLinkTemplate('remote-translation-review', '/translation-request/{oe_translation_request}/review');
+    }
+  }
+}
+
+/**
+ * Select list allowed options callback for the request statuses.
+ */
+function oe_translation_remote_request_statuses(FieldStorageConfig $definition, ContentEntityInterface $entity = NULL, bool $cacheable) {
+  $options = [
+    TranslationRequestRemoteInterface::STATUS_REQUEST_ACTIVE => t('Active'),
+    TranslationRequestRemoteInterface::STATUS_REQUEST_FINISHED => t('Finished'),
+  ];
+
+  // @todo Implement event subscriber to provide other options.
```

Comment:
We will do this when we create the epoetry submodule and we will need different request-level statuses in addition to these ones.